src/Pages/HelloWorld.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { HelloWorldPage } from "../Components";
+
+const HelloWorld: React.FC = () => {
+  return (
+    <HelloWorldPage />
+  )
```

Comment:
```suggestion
  return <HelloWorldPage />
```

추가할 사항이 없다면 이런 식으로 줄이는 것도 좋을 것 같습니다!